### ISSUE TITLE
Signal crashes sometimes while calling and the person doesn't answer

### ISSUE THREAD

**Issue by Dyras** [WhisperSystems/Signal-Android#5222]
Hi. Sometimes when I call someone and the person doesn't pick up, the app crashes. No idea why, but here are some logs.

https://gist.github.com/anonymous/c2b1eecfa72eb6a128e7
https://gist.github.com/anonymous/b83542637606292dbdc1
https://gist.github.com/anonymous/c0950a3a70d8f114dc9d

**Comment by cascheberg** [comment]
`java.lang.IllegalStateException at android.media.MediaPlayer._start(Native Method)`
Same in #4184.

**Comment by Dyras** [comment]
I'm not in airplane mode though. This happens on both 3G and Wi-Fi.

**Comment by peterpanondrugs** [comment]
https://gist.github.com/b2b3ab492f053bd7030c

**Comment by agrajaghh** [comment]
@peterpanondrugs waht exactly are you experiencing? You already posted the same debug log in #5216 (+ I don't see a crash in that log)

**Comment by peterpanondrugs** [comment]
https://gist.github.com/5c59d530ce2aeb6252c0

![screenshot_2016-02-12-09-54-47](https://cloud.githubusercontent.com/assets/8776733/13034005/4cee4b5e-d329-11e5-97b9-5867d98de7bb.png)

**Comment by agrajaghh** [comment]
@peterpanondrugs the blank/black screen looks like a duplicate of #5132

I still don't see a crash in that log (but maybe I'm looking for the wrong thing :smirk:). How long after the crash did you wait before you generated this debug log? And by "crash" you are not referring to the screenshot, but to the message "Unfortuantely Signal has stopped. [Report] [Ok]", right?

**Comment by peterpanondrugs** [comment]
signal def. crashed as i said. (Signal was closed, and it closed itselfs)
the black screen was still open after crash, but only after 1st crash. 
+ notification (still calling, but couldnt hang up)
what helped to get rid of the notification was starting a new call or restart system.

. 1st log: https://gist.github.com/b2b3ab492f053bd7030c <-- immediately after crash
.2nd log: https://gist.github.com/5c59d530ce2aeb6252c0 <-- half day later or smth 
so actually both should show the crash.

**Comment by cascheberg** [comment]
I think this issue has the same root cause as #4184. It might be occurring when playing a failure sound immediately before terminating. @moxie0 Please have a look at #4886 again.

@peterpanondrugs I'd also guess that your issue was #5216. Should be fixed in 3.11.1 which you seem to be using now, but not when the crashes happened.

**Comment by agrajaghh** [comment]
fixed by #5357
